README.md

```diff
@@ -148,3 +149,32 @@ git config --global core.autocrlf false
 ```bash
 git clone https://github.com/usdot-jpo-ode/jpo-cvdp.git
 ```
+
+# Confluent Cloud Integration
+Rather than using a local kafka instance, this project can utilize an instance of kafka hosted by Confluent Cloud via SASL.
+
+## Environment variables
+### Purpose & Usage
+- The DOCKER_HOST_IP environment variable is used to communicate with the bootstrap server that the instance of Kafka is running on.
+- The KAFKA_TYPE environment variable specifies what type of kafka connection will be attempted and is used to check if Confluent should be utilized.
+- The CONFLUENT_KEY and CONFLUENT_SECRET environment variables are used to authenticate with the bootstrap server.
+
+### Values
+- DOCKER_HOST_IP must be set to the bootstrap server address (excluding the port)
+- KAFKA_TYPE must be set to "CONFLUENT"
+- CONFLUENT_KEY must be set to the API key being utilized for CC
+- CONFLUENT_SECRET must be set to the API secret being utilized for CC
+
+## CC Docker Compose File
+There is a provided docker-compose file (docker-compose-confluent-cloud.yml) that passes the above environment variables into the container that gets created. Further, this file doesn't spin up a local kafka instance since it is not required.
+
+## Note
+This has only been tested with Confluent Cloud but technically all SASL authenticated Kafka brokers can be reached using this method.
+
+# Troubleshooting
+## Standalone Cluster
+The docker-compose-standalone.yml file is meant for local testing/troubleshooting. To utilize this, rename it to docker-compose.yml and run docker-compose up.
```

Comment:
Let's use the `docker-compose -f FILE` syntax instead of recommending they rename to docker-compose.yml since we already have a docker-compose.yml file in that directory.

Reply:
This section has been edited to suggest using that command as you recommended.